web/src/ProviderEditPage.js

```diff
@@ -1095,6 +1097,31 @@ class ProviderEditPage extends React.Component {
                   </Col>
                 </Row>
               )}
+              <Row style={{marginTop: "20px"}} >
+                <Col style={{marginTop: "5px"}} span={2}>
+                  {Setting.getLabel(i18next.t("general:Method"), i18next.t("provider:Method - Tooltip"))} :
+                </Col>
+                <Col span={22} >
+                  <Select virtual={false} style={{width: "100%"}} value={this.state.provider.method} onChange={value => {
+                    this.updateProviderField("method", value);
+                  }}>
+                    {
+                      [
+                        {id: "GET", name: "GET"},
```

Comment:
For HTTP method options, see:

https://github.com/casibase/casibase/blob/4d82c3e82c4443317a1b59045389715e711e24e1/web/src/RecordEditPage.js#L160-L170